tox.ini

```diff
@@ -0,0 +1,18 @@
+[tox]
+envlist = py27, py34, py35, py36, py37, py38
+
+[testenv]
+deps =
+    -r {toxinidir}/requirements.txt
+    -r {toxinidir}/tests/requirements.txt
+
+setenv =
+    PATH = {toxworkdir}/bin{:}{env:PATH}
+
+commands =
+    flake8 .
+    /bin/sh -c 'doc8 `git ls-files *.rst`'
+    /bin/sh -c 'yamllint --strict `git ls-files *.yaml *.yml`'
+    python setup.py test
+
+# vim:sw=4:ts=4:et:
```

Comment:
I suggest adding this line to your `~/.vimrc` or `~/.config/nvim/init.vim`:

```vim
autocmd FileType python setlocal expandtab tabstop=4 softtabstop=4 shiftwidth=4
```

Reply:
Oh, thanks for your info! I've not been updating my ~/.vim/ for a while but will try that.